.github/workflows/check-excluded-packages.yml

```diff
@@ -81,5 +81,19 @@ jobs:
             popd
           done
 
+      - name: Install nono
+        run: cargo install cargo-nono
+
+      - name: Assert that packages are compatible with no-std
+        run: |
+          packages=("relations" "bin/runtime/src/chain_extension/snarcos-extension")
```

Comment:
Does this extension have to be compatible with no-std or just wasm32-unknown-unknown?

Reply:
it is used by contracts, which are compiled with both `std` and `no-std`, so I suppose that it has to be